.github/workflows/repo-stale.yaml

```diff
@@ -0,0 +1,30 @@
+name: Issue Stale Check
+
+on:
+  schedule:
+    - cron: '30 1 * * *'
+  workflow_dispatch:
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
```

Comment:
The workflows in this repository use explicit versions, so this should be `ubuntu-20.04`.